sigmajs/sigmajs.d.ts

```diff
@@ -15,10 +15,17 @@ declare module SigmaJs{
         graphPosition(x: number, y:number): {x: number; y: number};
         ratio: number;
         readPrefix: string;
+        settings(setting: string) : any;
         x: number;
         y: number;
     }
 
+    interface Canvas {
+        edges: {[renderType: string]: Function};
+        labels: {[renderType: string]: Function};
+        nodes: {[renderType: string]: Function};
```

Comment:
Is it possible to be more specific about the `Function` types here? `Function` is a cop-out, similar to `any`.